pkg/connector/tickets.go

```diff
@@ -94,6 +94,9 @@ func (d *Connector) schemaForProject(ctx context.Context, project *jira.Project)
 	}
 
 	for _, issueType := range project.IssueTypes {
+		if issueType.Name == "Epic" || issueType.Name == "Bug" {
+			continue
+		}
```

Comment:
Maybe we could do the same thing as project filters, they provide us issue type keys to filter BUT this is fine for now